Development/src/pages/logs.js

```diff
@@ -200,7 +183,7 @@ const EventsShowActions = ({ basePath, data, resource }) => (
             </Button>
         ) : null}
         <ListButton title={'Return to ' + basePath} basePath={basePath} />
-    </CardActions>
+    </TopToolbar>
 );
 
 export const EventsShow = props => (
```

Comment:
May be worth refactoring logs into list and show view like other resources

Reply:
Yes, consistency would be nice.